### PR TITLE
JUSI-374 update jwst-cal s/w to 0.16.1 or later

### PR DESCRIPTION
**Pull request by jaytmiller**
Updated jwst-cal with CAL 0.16.1, MIRISIM 2.2.0, and MIRAGE 1.3.3 w/jwst-0.16.1.   DMS-7.5rc2.
Updated CAL example notebook image registration .cfg files using collect_pipeline_cfgs.
Added import checks for jwst-cal, mirisim, mirage.
CAL notebook runs locally with CRDS env changes,  data may have issues with expected units.
Image default env for CRDS works on AWS provided worker has CRDS read perms.
Added utils directory with login, build, run, push scripts driven by vars.sh files.
Refactored jwst-cal to a standard single-version directory layout like the other projects,  dropping 0.13.7 dedicated subdir.
Added an extra digit to jwst-cal VERSION to supplement the normal CAL release tag.

### CODE REVIEW

File: jwst-cal/home_template/example_notebooks/pipeline_img_registration_miri/outlier_detection.cfg [hunk 1]
```diff
@@ -14,4 +14,4 @@ scale = '0.5 0.4'
 backg = 0.0        
 save_intermediate_results = False
 resample_data = True
-good_bits = 4
+good_bits = '~DO_NOT_USE'
```
Comment (cslocum) [inline jwst-cal/home_template/example_notebooks/pipeline_img_registration_miri/outlier_detection.cfg:17]: What is this used for?

**Comment by cslocum** [comment]
@jaytmiller This looks great!  And thanks for the README.

I did leave one question, but it's not all that important.

**Comment by jaytmiller** [comment]
XXXX

This PR should NOT be merged as-is.   The real requirement is to do similar updates to  the jwst-masterclass image,   works still incomplete.

**Comment by jaytmiller** [comment]
Withdrawn,  switch to jwst-masterclass.

**Pull request closed without merging**